test/library/packages/Yaml/deserializeByRef.good

```diff
@@ -1,5 +1,5 @@
 ===== writing: =====
-(values = [(x = 1, b = (t = (11, 1.1))), (x = 2, b = (t = (22, 2.2))), (x = 3, b = (t = (33, 3.3)))])
+[(x = 1, b = (t = (11, 1.1))), (x = 2, b = (t = (22, 2.2))), (x = 3, b = (t = (33, 3.3)))]
```

Comment:
I'm not seeing where this change came from. Is `values` not being named a side effect of the `_firstPtr` changes since it's the only field in `A`?

Reply:
This is actually because the relevant writeln was using ``writeThis`` before, but now it actually invokes ``serialize`` which is defined to only print the values of ``values``, and not a field. It's not related to the firstPtr stuff.

Reply:
Okay that makes sense. Thanks!